src/platform/update/version.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2019 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MULTIPASS_VERSION_COMPARITOR_H
+#define MULTIPASS_VERSION_COMPARITOR_H
+
+#include <QString>
+// glibc defines major & minor macros which interfere with the class method names
```

Comment:
But then you are removing them for anyone who includes this file. I would prefer using different names if these are reserved (e.g. appending `_version`)

Reply:
file removed